e2e/browser/testSuiteRunner.test.ts

```diff
@@ -172,3 +181,11 @@ testSuite.podServerList.forEach((server: IPodServerConfig) => {
     });
   }
 });
+
+// it("is still logged in after refresh", async () => {
```

Comment:
Not sure what's going on here - was this just left behind from some informal testing you were doing...?

Reply:
That's absolutely what's going on, leftover test code. Sorry for the noise, I usually try to go through the code before asking for reviews, but I was waiting on the e2e tests to go through first :)